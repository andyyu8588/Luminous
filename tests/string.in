print("Hello " + "world!");
print("Andy " + "is " + "pog.");
print("And Yunze is awesome!");
print("Yo " + "I'm " + "cool?" equals "Yo I'm cool?");
print("G fucking G!" equals("G" + "fucking" + "G!"));