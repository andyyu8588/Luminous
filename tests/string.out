Hello world!
Andy is pog.
And Yunze is awesome!
true
false
